app/config/sources.inc.php

```diff
@@ -416,6 +416,11 @@
             'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
         ],
     ],
+    'mozorg/about/governance/policies/community-hotline.lang' => [
+        'supported_locales' => [
+            'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
```

Comment:
per bug https://bugzilla.mozilla.org/show_bug.cgi?id=1600148#c8 - the locale list should expand to 

 ar, de, es-ES, fr,  hi, id, it, ja, nl, pl, pt-BR, ru,  zh-CN, zh-TW

This means all three CPG files need to be updated with the locale list

Reply:
Ah, I missed that. Thanks!